CHANGELOG.md

```diff
@@ -4,4 +4,4 @@ All notable changes to the Zowe Installer will be documented in this file.
 
 ## Recent Changes
 
-- When `localhost` cannot be resolved use the IP address instead.
\ No newline at end of file
+- When the hostname cannot be resolved use the IP address instead.  This covers the scenarios when `localhost` was not specified as an alias for the loopback address 127.0.0.1 as well as when the USS `hostname` command returned a system name that wasn't externally addressable, such as `S0W1.DAL-EBIS.IHOST.COM` which occurs on an image created from the z/OS Application Developers Controlled Distribution (ADCD).
```

Comment:
AFAIK, localhost is not involved anywhere in this check in `zowe-init.sh`, so the wording 

> s when `localhost` was not specified as an alias for the loopback address 127.0.0.1 as well as

should be removed.

Reply:
Tks @John-A-Davies - I've pushed an updated to match the wording correction.